README.md

```diff
@@ -1,328 +1,126 @@
-|                         Shards aggregate chunks within individual files or S3 buckets.                          |
-
-Suppose you have a video stream with the following dimensions:
-
-- Width: 1920 px
-- Height: 1080 px
-- Channels: 3
-- Time: 1000 frames (per channel)
-
-You want to divide your frames into 4 x 4 tiles of size 480 x 270, and you want each channel to be stored in separate
-chunks.
-You also want to aggregate 100 time points into a single chunk.
-You will end up with 4 x 4 = 16 chunks of size 480 x 270 x 1 x 100 for each channel, for a total of 48 chunks, for every
-100 time points.
-Suppose further that you wanted to aggregate each of these chunks into a single *shard*.
-Configuring your storage dimensions might look like this:
-
-```cpp
-struct StorageProperties props = { 0 };
-
-const char filename[] = "my_video.zarr";
-const char external_metadata[] = R"({"my":"metadata"})";
-const struct PixelScale sample_spacing_um = { 1, 1 };
-storage_properties_init(&props,
-                        0, // first frame id
-                        (char*)filename,
-                        strlen(filename) + 1,
-                        (char*)external_metadata,
-                        sizeof(external_metadata),
-                        sample_spacing_um,
-                        4); // number of dimensions
-
-// width
-storage_properties_set_dimension(
-  &props,
-  0,
-  "x", // name
-  2,   // number of bytes in the name, including the null terminator
-  DimensionType_Space, // type of the dimension
-  1920,                // full size of the dimension
-  480,                 // number of pixels in a chunk
-  4); // aggregate all 4 chunks into a shard along this dimension
-
-// height
-storage_properties_set_dimension(
-  &props,
-  1,
-  "y", // name
-  2,   // number of bytes in the name, including the null terminator
-  DimensionType_Space, // type of the dimension
-  1080,                // full size of the dimension
-  270,                 // number of pixels in a chunk
-  4); // aggregate all 4 chunks into a shard along this dimension
-
-// channels
-storage_properties_set_dimension(
-  &props,
-  2
-  "c", // name
-  2,   // number of bytes in the name, including the null terminator
-  DimensionType_Channel, // type of the dimension
-  3,                     // full size of the dimension
-  1,                     // number of pixels in a chunk (1 channel per chunk)
-  1); // one single-channel chunk per shard along this dimension
-
-// time
-storage_properties_set_dimension(
-  &props,
-  3,
-  "t", // name
-  2,   // number of bytes in the name, including the null terminator
-  DimensionType_Time, // type of the dimension
-  0,   // append dimension; we don't know the full size a priori
-  100, // number of pixels in a chunk
-  1);  // one 100-timepoint chunk per shard along this dimension
+```bash
+cmake --build /path/to/build
 ```
 
-### Compression
+## Usage
```

Comment:
Is there an example for the python api somewhere?

Reply:
There is now.